src/Microsoft.ML.Transforms/Text/TextFeaturizingEstimator.cs

```diff
@@ -120,26 +120,59 @@ internal sealed class Arguments : TransformInputBase
             public TextNormKind VectorNormalizer = TextNormKind.L2;
         }
 
-        public sealed class Settings
+        /// <summary>
+        /// Advanced settings for the <see cref="TextFeaturizingEstimator"/>.
```

Comment:
>settings [](start = 21, length = 8)

can we call it options?

Reply:
fixed. 

am planning to have another PR which fixes this for all the learners/featurizers we  converted to Options  

---
In reply to: [254802978](https://github.com/dotnet/machinelearning/pull/2394#discussion_r254802978) [](ancestors = 254802978)